spec/factories/users.rb

```diff
@@ -16,11 +16,13 @@
 
   trait :supervisor do
     supervisor true
+    staff false
     badge_number nil
   end
 
   trait :staff do
     staff true
+    supervisor { rand(2).zero? }
```

Comment:
Open to more readable suggestions. Perhaps `[true, false].sample`?

Reply:
My vote is for `[true, false].sample`. But the line you already have makes sense, too. As long other programmers can tell `supervisor` is a boolean.

Reply:
https://github.com/umts/incidents/pull/83/commits/3089373886a3361d4b2065d30d150489b746bdb8